# Generated for NaoQi method ALLandMarkDetection.run
# protected method

---


